README.md

```diff
@@ -4,4 +4,4 @@ Esta es una descripcion breve del proyecto
 
 ## Objetivo
 
-Practicar las pr 😃
\ No newline at end of file
+Practicar las pr 😃. Estoy probando otra cosa 🤓
```

Comment:
Deberias modificarlo por 'Practica de PR'. Si podes cambiame los iconos que no me gustan 😢

Reply:
i already changed the file 😎